### PR TITLE
Misc Quest Fixes and Cleanup

### PR DESCRIPTION
**Pull request by IntegerLimit**
This PR contains more miscellaneous QB fixes and cleanup. This includes:
- Cleans Up Platinum Group Processing quest, Allow Using Tier Three Micro Miner for Initial Ruthenium
- Improvement in Air Distillation Quests
	- Adds Nitrogen and Basic Air Collection Quests to The Beginning
	- Removes Nitrogen from Mid Game
	- Deletes Air Centrifuging Quest
- General Improvement in Clarity of Quests
- Cleanup of Mid Game and The Beginning Lines

### CODE REVIEW

File: overrides/config-overrides/expert/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -2297,7 +2297,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "This alloy is made from §2mixing §22 Gold Dust, 1 Redstone, and 1 Glowstone Dust §2and putting it §rin your new §3Electric Blast Furnace§r.\n\nThrow §2the prepared dust into the input bus§r and in twenty seconds your §6Energetic Alloy Ingot§r will be in the output bus.\n\n§2In EV Age, you can build an Alloy Blast Smelter which doubles the yield.",
+          "desc:8": "This alloy is made from §2mixing §22 Gold Dust, 1 Redstone, and 1 Glowstone Dust §2and putting it §rin your new §3Electric Blast Furnace (EBF)§r.\n\nThrow §2the prepared dust into the input bus§r and in twenty seconds your §6Energetic Alloy Ingot§r will be in the output bus.\n\nYou might want to consider getting some §9Nitrogen§r, which speeds up EBF recipes. See the \"Basic Air Collection\" quest to start your journey. For this recipe, it reduces the duration to 13.7 seconds.\n\n§2In EV Age, you can build an Alloy Blast Smelter which doubles the yield.",
```
Comment (Doniazade) [inline overrides/config-overrides/expert/betterquesting/DefaultQuests.json:2300]: Comment unrelated to changes in this PR: Are we referring to voltages as ages? I don't remember seeing this a lot.

**Review by Doniazade — APPROVED** [review]
Reviewed based on comments and code. At work.

Good changes. The main issue with how it is in the QB on live is that while platinum group processing is satisfied by the ruthenium from t3mm, the quest doesn't guide the player to t3mm to get it, leading some players to do an early and tedious platline for no reason. So maybe also add a comment to platinum group processing summarizing the options for getting platline metals from mms, especially ruthenium from t3mm laurite.

Methane water split (ch4 + 2 h2o -> 8 h + co2) is HV, but could maybe be mentioned along with methane electrolysis so the player is aware that it's a superior upcoming option.

Changes are good to go as it is however.

File: overrides/config-overrides/expert/betterquesting/DefaultQuests.json (hunk 1 above)
Comment (IntegerLimit) [reply overrides/config-overrides/expert/betterquesting/DefaultQuests.json:2300]: No we are not, let me fix that, as well as try to guide the player towards t3mm more in PGS.
